spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
In fact, it's the `escape_html` that causes the problem. Removing `CGI::escape_html` lets the test pass.

Why are we using Minitest assertions in our RSpec suite?